core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -254,25 +254,26 @@ public BuddyDto addBuddyToAcceptingUser(UserDto acceptingUser, BuddyConnectReque
 	@Transactional
 	public void removeBuddyAfterConnectRejection(UUID idOfRequestingUser, UUID buddyId)
 	{
-		User user = userService.getValidatedUserById(idOfRequestingUser);
+		userService.getValidatedUserById(idOfRequestingUser);
```

Comment:
It would be more speaking to just call a `validate` method instead of a `get`.

Reply:
Done.